From 7be1b81102e71617ddc568f7957ccc1429f0513d Mon Sep 17 00:00:00 2001
From: magic_rb <richard@brezak.sk>
Date: Mon, 15 Jul 2024 15:44:15 +0200
Subject: [PATCH 2/3] `GiteaHandler`, set `branch` to the PR branch not the
 base

Signed-off-by: magic_rb <richard@brezak.sk>
---
 buildbot_gitea/webhook.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/buildbot_gitea/webhook.py b/buildbot_gitea/webhook.py
index a63cfad..17d4809 100644
--- a/buildbot_gitea/webhook.py
+++ b/buildbot_gitea/webhook.py
@@ -97,9 +97,9 @@ class GiteaHandler(BaseHookHandler):
                 pull_request['number'],
                 pull_request['title'],
                 pull_request['body']),
-            'revision': base['sha'],
+            'revision': head['sha'],
             'when_timestamp': timestamp,
-            'branch': base['ref'],
+            'branch': head['ref'],
             'revlink': pull_request['html_url'],
             'repository': base['repo']['ssh_url'],
             'project': repository['full_name'],
-- 
2.44.1

